_develop/mining_enroll.md

```diff
@@ -17,74 +17,87 @@ rewards. This section explains the enrollment process and covers the following t
 
 ## Complete the add and submission forms on App.co
 
-Make sure you enroll in advance of the month you would first like your app to complete.  You need only enroll in App Mining once.  A single submission enrolls your app in the month you submit and every month following automatically.
+Make sure you enroll in advance of the month you would first like your app to complete.  You need only enroll in App Mining once.  A single submission enrolls your app in the month you submit and every month following automatically. You can submit multiple applications.
 
-To apply, do the following:
+To submit an application, do the following:
 
-1. Integrate Blockstack Auth into your app.
+1. Create an application that integrates Blockstack Auth.
 
    Your application code can reside in a public or a private repository.
 
-2. Complete the  <a href="https://app.co/submit" target="\_blank">App.co</a> form.
+2. Choose <a href="https://app.co/submit" target="\_blank"><strong>Submit app</strong></a> from the App.co site.
 
-   You should fill out all the required fields. 
+3. Sign into the submission form with your Blockstack ID.
 
-   <img src="images/add-app.png" alt="">
+   <img src="images/submission-signin.png" alt="">
+
+   You will use your Blockstack ID to access your submission in the future. If you have multiple IDs, make sure you keep track of which ID you use during your submission
 
+4. Fill out all the form's required fields. 
+
+   <img src="images/add-app.png" alt="">
 
    {% include required-fields.md %}
    
-   Make sure you choose **Blockstack** from the **Authentication** list on the form.
+   You must choose **Blockstack** from the **Authentication** category on the form; If you do not, you cannot participate in App Mining.
 
+5. Check the box that says *App is publicly accessible and user-ready(required)*.
+6. Agree to the terms and policies.
 
-3. Press **Submit App**.
+7. Press **Submit your app**.
 
-   The system responds with the a completion dialog and an application link. 
+   The system responds with the a completion dialog. 
 
    <img src="images/app-submission-success.png" alt="">
    
    You will also receive an email from the App.co team acknowledging receipt of your submission.
 
-4. Copy and store the application link.
+8. Click **Participate in App Mining** to view a list of your submissions.
```

Comment:
@kyranjamie Could we make the "Participate in App Mining" button here on the app submission confirmation page take the user directly to the manage page for the app they just submitted, instead of the directory of apps?

Reply:
Done.

Reply:
Thanks! @moxiegirl Want to modify this text to reflect this change?

Reply:
Sure thing. I'll do that!

Reply:
@markmhx @kyranjamie  maybe after pressing submit they go directly to the manage page? Is that what you implemented @kyranjamie?

Reply:
@markmhx shouldn't *Participate in App Mining* be *Manage your apps* --> that will also teach them what to look for on the menu.

Reply:
@moxiegirl, nope. Clicking the button now takes them directly to the `/maker/apps/<new_app_id>` page.

Reply:
As @kyranjamie indicates, with the user taken to the exact app's "manage" page (instead of the directory of apps to manage a la "Manage your apps"), this copy would be more accurate as:

 "Click **Participate in App Mining** to submit the information necessary for enrollment in App Mining" or something similar

Reply:
@markmhx @kyranjamie please see https://github.com/blockstack/app.co/issues/751

Reply:
Thanks – I just commented there with a proposal